packages/angular/src/generators/convert-tslint-to-eslint/convert-tslint-to-eslint.ts

```diff
@@ -27,6 +27,7 @@ export async function conversionGenerator(
   const projectConverter = new ProjectConverter({
     host,
     projectName: options.project,
+    discardExistingLintConfig: options.discardExistingLintConfig,
     eslintInitializer: async ({ projectName, projectConfig }) => {
       await addLintingGenerator(host, {
```

Comment:
I didn't have `eslint` installed after converting the first project.

This won't return a generator callback because it's a wrapped angular devkit schematic.. I think we can assume there should be an install task and return an install task.

Reply:
I _think_ I addressed your feedback here